accessibility_monitoring_platform/apps/reports/utils.py

```diff
@@ -267,3 +268,11 @@ def get_report_viewer_url_prefix(request: HttpRequest) -> str:
             return ".".join(domain_name_split)
     else:
         return ""
+
+
+def get_report_visits_metrics(case_model: Case) -> Dict[str, str]:
```

Comment:
The usual naming convention is to call an instance of the model by the model name converted into lower snake case. In this case, `case`.